### PR TITLE
Adminbus Things

### PR DESCRIPTION
**Pull request by Geevies**
* Adds full variants of stacks of items in the spawn menu.
* Adds local narrate, which allows you to narrate to all mobs in a 7 tile radius.
* Allows admins to reset stationbounds via right click or Special Verbs menu.
* Adds a random type golem rune to the spawn menu, if an admin wants to adminbus a random golem into a round.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -118,6 +118,23 @@
 	message_admins("<span class='notice'>\bold GlobalNarrate: [key_name_admin(usr)] : [msg]<BR></span>", 1)
 	feedback_add_details("admin_verb","GLN") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_local_narrate()
+	set category = "Special Verbs"
+	set name = "Local Narrate"
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
```
Comment (NonQueueingMatt) [inline code/modules/admin/verbs/randomverbs.dm:126]: this does not actually check for any specific rights though

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -118,6 +118,23 @@
...
+		return
+
+	var/msg = html_decode(sanitize(input("Message:", text("Enter the text you wish to appear to everyone within seven tiles of you:")) as text))
+	if(!msg)
+		return
+	for(var/mob/M in range(7, src.mob))
+		to_chat(M, msg)
```
Comment (NonQueueingMatt) [inline code/modules/admin/verbs/randomverbs.dm:133]: This should really just be a visible message or at least a choice between one and an ignore-all message

File: code/modules/admin/verbs/randomverbs.dm [hunk 3]
```diff
@@ -296,6 +313,30 @@ proc/get_ghosts(var/notify = 0,var/what = 2, var/client/C = null)
 	log_admin("[key_name(usr)] allowed [key_name(G)] to bypass the 30 minute respawn limit",admin_key=key_name(usr),ckey=key_name(G))
 	message_admins("Admin [key_name_admin(usr)] allowed [key_name_admin(G)] to bypass the 30 minute respawn limit", 1)
 
+/client/proc/allow_stationbound_reset(mob/living/silicon/robot/R in range(world.view))
+	set category = "Special Verbs"
+	set name = "Allow Stationbound Reset"
+	set desc = "Select a stationbound to reset its module."
+
+	if(!holder)
```
Comment (NonQueueingMatt) [inline code/modules/admin/verbs/randomverbs.dm:321]: ditto

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (Geevies) [reply code/modules/admin/verbs/randomverbs.dm:126]: holder is a admin holder thing, and all the other admin verbs make use of it, so i assume it's fine? i can make it more robust if i need to

Comment (Arrow768) [reply code/modules/admin/verbs/randomverbs.dm:126]: Holder is just the general permissions holder.
-> Everyone with some sort of permission has that (Admins, Mods, Devs, CCIA, ...)

File: code/modules/admin/verbs/randomverbs.dm [hunk 4]
```diff
@@ -118,6 +118,33 @@
...
+	if(!check_rights(R_ADMIN, TRUE))
+		return
+	
+	var/list/mob/message_mobs = list()
+	var/choice = alert(usr, "Local narrate will send a plain message to mobs. Do you want the mobs messaged to be only ones that you can see, or ignore blocked vision and message everyone within seven tiles of you?", "Narrate Selection", "In my view", "In range of me", "Cancel")
+	if(choice != "Cancel")
+		if(choice == "In my view")
+			message_mobs = mobs_in_view(7, src.mob)
+		else
+			for(var/mob/M in range(7, src.mob))
```
Comment (Arrow768) [inline code/modules/admin/verbs/randomverbs.dm:134]: It´s possible to change the view range for admins, so that would imho be useful to integrate here.

Comment (Geevies) [reply code/modules/admin/verbs/randomverbs.dm]: done

Comment (Geevies) [reply code/modules/admin/verbs/randomverbs.dm]: i think

**Pull request merged**